components/ParameterForm.vue

```diff
@@ -390,4 +425,52 @@ onMounted(() => {
     box-shadow: 0 0 0 15px rgba(0, 0, 255, 0);
   }
 }
+
+.select-container {
+   margin-left: 0.7rem;
+   margin-right: 0.55rem;
+}
+
+.numeric-header {
+  padding-right: 2.2rem;
+}
+
+.vue-select {
```

Comment:
should this vue selector styles be in globals? since any future selects should look the same

Reply:
Ooh good point  - yes, possibly. But  I think these styles are quite specific to the context of the parameter form, so I think it's ok for them to stay here for now. If we ever do add any more vue selects, we'll split out what can go be shared in globals then I think.